packages/attendance/src/utils/functions/GetStudentsAttendance.ts

```diff
@@ -0,0 +1,40 @@
+import moment from "moment";
+
+export interface IGetStudentsAttendance {
+  students?: Array<any>;
+  attendance: Array<any>;
+  type?: string;
+}
+
+// Returns attendance following a certain criteria
+// type specifies on what criteria
+export const GetStudentsAttendance = ({
+  students = [],
+  attendance,
+  type = "format",
+}: IGetStudentsAttendance) => {
+  if (type.toLowerCase() === "format")
+    return students
+      .map((item) => {
+        return attendance
+          .slice()
+          .reverse()
+          .find(
+            (e) =>
+              e.date === moment().format("YYYY-MM-DD") &&
+              e.studentId === item.id
+          );
+      })
+      .filter((e) => e);
```

Comment:
why this is required?

Reply:
I added it since we use a similar functionality in one of the other components, but for now I'll only add the required functionality.